docs/source/run/parameters.rst

```diff
@@ -193,6 +193,12 @@ General parameters
     Currently, this option only affects plasma operations (gather, push and deposition).
     The tile size can be set with ``plasmas.sort_bin_size``.
 
+* ``hipace.comms_buffer_on_gpu`` (`bool`) optional (default `false`)
```

Comment:
We usually use `0` and `1` for booleans in the input file, could you make this consistent here?
```suggestion
* ``hipace.comms_buffer_on_gpu`` (`bool`) optional (default `0`)
```